src/test/java/no/nav/pto/veilarbportefolje/opensearch/OpensearchQueryBuilderTest.java

```diff
@@ -38,6 +38,13 @@ public void skal_sortere_etternavn_paa_etternavn_feltet() {
         assertThat(fieldName).contains("etternavn");
     }
 
+    @Test
+    public void skal_sortere_pa_neste_svarfrist_stilling_fra_nav() {
+        var searchSourceBuilder = sorterQueryParametere("asc", "neste_svarfrist_stilling_fra_nav", new SearchSourceBuilder(), new Filtervalg());
+        var fieldName = searchSourceBuilder.sorts().get(0).toString();
+        assertThat(fieldName).contains("neste_svarfrist_stilling_fra_nav");
+    }
+
```

Comment:
Er dette en test som egentlig er noe verdifull? Burde jeg heller lagd en test for å sortere noen datoer? Jeg gjorde bare testen for å sjekke at den fant feltet

Reply:
Testen er grei i forhold til at man tester `OpensearchQueryBuilder.sorterQueryParametere`, men så vidt jeg ser tester den akkurat det samme som testen over på etternavn, som ikke er spesialhåndtert og ender med å bruke metoden `OpensearchQueryBuilder.defaultSort`